plugins/core/IO/qLASIO/include/LasSaver.h

```diff
@@ -40,14 +40,15 @@ class LasSaver
 		std::vector<LasExtraScalarField> extraFields;
 		bool                             shouldSaveRGB{false};
 		bool                             shouldSaveWaveform{false};
+		bool                             shouldSaveNormalsAsExtraScalarField{false};
 		uint8_t                          versionMajor{1};
 		uint8_t                          versionMinor{0};
 		uint8_t                          pointFormat{0};
 		CCVector3d                       lasScale;
 		CCVector3d                       lasOffset;
 	};
 
-	LasSaver(ccPointCloud& cloud, Parameters& parameters);
+	LasSaver(ccPointCloud& cloud, Parameters parameters);
```

Comment:
Or maybe:
```suggestion
	LasSaver(ccPointCloud& cloud, const Parameters& parameters);
```

Reply:
Hum can't do that because in the implementation whe modifiy it if parameters.shouldSaveNormalsAsExtraScalarField is true.

if we take a reference, we would need to temporarily copy in certain case